packages/data-layer/src/queries.ts

```diff
@@ -312,17 +307,12 @@ export const getRoundByIdAndChainId = gql`
   }
 `;
 
-export const getRoundsByProgramIdAndUserAddress = gql`
-  query getRoundsByProgramIdAndMemberAddress(
-    $chainId: Int!
-    $programId: String!
-    $userAddress: String!
-  ) {
+export const getRoundsByProgramId = gql`
```

Comment:
```suggestion
export const getRoundsByProgramIdAndChainId = gql`
```
If we are adding a chainId filter, would recommend updating this to getRoundsByProgramIdAndChainId as in v2 we can have rounds across chains by the same program/profile

Reply:
Would recommend doing the same for the above 2 queries : getProgramByIdAndUser, getProgramByUserAndTag to make it easier to read

Reply:
I think it's a good point but this PR is not adding a chain ID filter, it's just removing the roles filter, renaming other queries is out of scope of this PR, we can do it in a separate one

Reply:
I updated the one I changed though since I am already renaming it, thanks for the suggestion!